README.md

```diff
@@ -37,24 +37,36 @@ Will result into this HTML (open graph metas are using property automatically) :
 
 Title & metas are set only on `didTransition` in case the route would be canceled.
 
+**Note**: og tags will not correctly work without using FastBoot or some sort of server side rendering.
```

Comment:
for bots not fully executing JavaScript (Google does).

https://webmasters.googleblog.com/2015/10/deprecating-our-ajax-crawling-scheme.html